src/python_interpreter.rs

```diff
@@ -22,6 +22,10 @@ print(json.dumps({
     "major": sys.version_info.major,
     "minor": sys.version_info.minor,
     "abiflags": sysconfig.get_config_var("ABIFLAGS"),
+    # implementation_lower is only specified by pypy
+    "interpreter": sysconfig.get_config_var("implementation_lower") or "cpython",
+    "ext_suffix": sysconfig.get_config_var("EXT_SUFFIX"),
+    "abi_tag": (sysconfig.get_config_var("SOABI") or "-").split("-")[1] or None,
```

Comment:
This can possibly replace the ABI flag logic, but I haven't tried python2.

Reply:
I'll drop python2 support anyway, so that's not problem